tests/integration/istioio/configuration/scripts/liveness_and_readiness_probes_with_command.txt

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+$ kubectl create ns istio-io-health
+
+# $snippet auth_policy.sh syntax="bash"
+$ kubectl apply -f - <<EOF
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "default"
+  namespace: "istio-io-health"
```

Comment:
Can you open a matching PR for the namespace changes on the istio.io docs?

Reply:
Sure, I will do that.

Reply:
Created matching PR for namespace change.
https://github.com/istio/istio.io/pull/6350

Reply:
Thanks